smac/facade/smac_facade.py

```diff
@@ -188,15 +220,29 @@ def __init__(self,
             self.scenario.transform_y = "LOG"
 
         # initialize empty runhistory
+        runhistory_def_kwargs = {'aggregate_func': aggregate_func}
+        runhistory_def_kwargs.update(runhistory_kwargs)
```

Comment:
And here you'd only have to add an `if runhistory_kwargs:` to check if the update is legal.